### PR TITLE
Fixed typos, improved language.

### PR DESCRIPTION
**Pull request by tladesignz**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -58,23 +58,23 @@ This tool is derived from [Cronet](https://chromium.googlesource.com/chromium/sr
...
-As earlier said it can be used to make app resistant to **censorship**.
+As earlier said, it can be used to make an app resistant to **censorship**.
 
-Also it can be used as proxy tool communication for secure web access to your app.
-E.g. if facebook and YouTube is not accessible to country like china and you want to make app which shows facebook content or YouTube content then you can use this tool to show your content.
+Also, it can be used as a proxy tool communication for secure web access to your app.
+E.g. if Facebook and YouTube are not accessible in countries like China, and you want to make an app which shows Facebook or YouTube content, then you can use this tool to show your content.
 
-It can be used as partial content showing or can be used to make whole server communication with this tool depending upon your requirement.
+It can be used for showing partial content, or it can be used to make all server communications depending upon your requirements.
```
Comment (mnbogner) [inline README.md:75]: Maybe "can be used to handle all server communications" makes more sense?

**Comment by mnbogner** [comment]
I'm not sure we ever spent much time on the top-level readme so thanks.  Aside from one minor nit-pick this looks fine to me.

**Comment by tladesignz** [comment]
I suggest to just merge this and then continue to improve the text with your own commits.

I'm not a native speaker, so for sure there are more things to find. I just stumbled over the most obvious, while reading through it. Hence the PR.

**Comment by stevenmcdonald** [comment]
Yes, thanks, having a new person looking at the project is always a good opportunity to update the docs. Fresh eyes and all that :)

**Pull request merged**